### ISSUE TITLE
re: CentOS 7.3,  vagrant up hangs on SSH step

### ISSUE THREAD

**Issue by MJMoody** [grtjn/mlvagrant#124]
With setting `vm_version=7.3`

Run `vagrant up --no-provision`

The above command hangs completely at the SSH step...

```
==> ml9-dhf-ml1: Waiting for machine to boot. This may take a few minutes...
    ml9-dhf-ml1: SSH address: 127.0.0.1:2222
    ml9-dhf-ml1: SSH username: vagrant
    ml9-dhf-ml1: SSH auth method: private key
```

When setting `vm_version=7.2`, the `vagrant up` process completes.

I am using the latest versions: 

VirtualBox 5.2.8
Vagrant 2.0.3

**Comment by grtjn** [comment]
I am aware of 7.3 not working well yet. I think the chef project made changes to how they put together their vm's causing this issue. I'll have a look if they have released a new version of the 7.3 vm, and/or if they released 7.4 and 7.5 versions too..

**Comment by grtjn** [comment]
See also #116

**Comment by grtjn** [comment]
i published a more recent 7.3 version, and also a 7.4 one, but neither works. The preview screen in virtual box shows a blinking login screen. It is as if the ssh auth is simply never reached for whatever reason.

**Comment by grtjn** [comment]
Still on-going. For the time being I recommend using the centos-7.2 (which is still the default for mlvagrant) for now. You can optionally update the os inside the vm to get it upgraded to 7.5 effectively. There is a properties flag to do so..

**Comment by grtjn** [comment]
I think I finally resolved this issue. It seems that re-packaging the bento base boxes messed up with its network config, causing the centos network service to fail, with subsequent timeouts when trying to connect from outside.

I found a fix for the re-packaging (mostly by sheer luck), and have in the meantime uploaded fresh versions of grtjn/centos-7.3 and grtjn/centos-7.4.

I recommend anyone that may have touched those base boxes to check and purge existing ones:

- vagrant box list (to find out if you have used either before)
- vagrant box remove grtjn/centos-7.3 (to get rid of it)
- vagrant box remove grtjn/centos-7.4

This is particularly necessary if you tried those boxes in the past few days, where I have been uploading 2017 and 2018 versions of both, and replacing them with new uploads..
